app/src/androidTest/java/com/github/orkest/navigationTabTest.kt

```diff
@@ -18,7 +19,7 @@ class navigationTabTest {
     @Test
     fun canAccessEveryNavigationTabFromMainActivity(){
         composeTestRule.setContent {
-            NavigationBar.CreateNavigationBar(navController = rememberNavController(),"")
+            NavigationBar.CreateNavigationBar(navController = rememberNavController(),"", MainActivity())
```

Comment:
This additional button is temporary while waiting for the picture functionality right?

Reply:
Yes exactly, the goal would be that when we would get a button to access the camera which at the same time launches the shazaming (which is also why I chose to use toasts messages to display if a match happened or not ..)